content/en/synthetics/testing_tunnel.md

```diff
@@ -0,0 +1,89 @@
+
+```sh
+datadog-ci synthetics run-tests --config <GLOBAL_CONFIG_FILE>.json --tunnel
+```
+
+### Firewall Requirements
+
+**Outbound connections** need to be allowed for the following Datadog endpoints:
+
+{{< site-region region="us" >}}
+
+| Port | Endpoint                                                                                             | Description                                                                                                                             |
+| ---- | ---------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| 443  | `tunnel-us1.synthetics.datadoghq.com`   | Required to open the wss connection from the `datadog-ci` client to the tunnel service. |
+| 443  | `intake.synthetics.datadoghq.com` | Required to get the presigned URL & to trigger the Synthetic tests. |
+| 443  | `api.datadoghq.com` | Required to search for Synthetic tests, get them, and poll their results. |
+
+{{< /site-region >}}
+
+{{< site-region region="eu" >}}
+
+| Port | Endpoint                                                                                             | Description                                                                                                                             |
+| ---- | ---------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| 443  | `tunnel-eu1.synthetics.datadoghq.com`   | Required to open the wss connection from the `datadog-ci` client to the tunnel service. |
+| 443  | `api.datadoghq.eu` | Required to get the presigned URL, search for Synthetic tests, get them, trigger them, and poll their results. |
+
+**Note**: Although the tunnel service top level domain is `.com` (and not `.eu`), the endpoint is located in EU (Frankfurt AWS).
+
+{{< /site-region >}}
+
+{{< site-region region="us3" >}}
+
+| Port | Endpoint                                                                                             | Description                                                                                                                             |
+| ---- | ---------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| 443  | `tunnel-us3.synthetics.datadoghq.com`   | Required to open the wss connection from the `datadog-ci` client to the tunnel service. |
+| 443  | `api.datadoghq.com` | Required to get the presigned URL, search for Synthetic tests, get them, trigger them, and poll their results. |
+
+{{< /site-region >}}
+
```

Comment:
what about gov/us1-fed? if this feature isn't available for that site, we might want to note that.